spec/models/post_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'validations' do
+    first_user = User.create(name: 'Johnie', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Tech enthusiast.',
+                             posts_counter: 1)
+    first_post = Post.create(author_id: first_user.id, title: 'Hello', text: 'This is my first post',
+                             comments_counter: 0, likes_counter: 0)
+
```

Comment:
- [ ] Kindly include some positive test case also. for ex:- 
- presennce of `title` should be `valid`.